src/main/java/seedu/eduke8/option/OptionList.java

```diff
@@ -1,27 +1,48 @@
 package seedu.eduke8.option;
 
+import seedu.eduke8.common.Displayable;
+import seedu.eduke8.common.DisplayableList;
+
 import java.util.ArrayList;
 
-public class OptionList implements OptionListInterface {
-    private final ArrayList<OptionInterface> options;
+public class OptionList implements DisplayableList {
+    private final ArrayList<Displayable> options;
 
     public OptionList() {
         options = new ArrayList<>();
     }
 
-    public OptionList(ArrayList<OptionInterface> optionList) {
-        this.options = optionList;
+    public OptionList(ArrayList<Displayable> options) {
+        this.options = options;
     }
 
-    public void addOption(OptionInterface option) {
-        options.add(option);
+    @Override
+    public ArrayList<Displayable> getInnerList() {
+        return this.options;
     }
 
-    public void deleteOption(OptionInterface option) {
+    @Override
+    public void add(Displayable option) {
         options.remove(option);
```

Comment:
I'm a bit confused with this. It's a add method but it removes the option...

Reply:
I think this should be add instead too!

Reply:
Yes sorry thanks, need to get out tests up quickly to catch these mistakes